core/domain/classifier_services.py

```diff
@@ -359,9 +458,8 @@ def delete_classifier_training_job(job_id):
         classifier_training_job_model.delete()
 
 
-def get_classifier_from_exploration_attributes(exp_id, exp_version,
-                                               state_name):
-    """Gets the classifier object from the exploration attributes.
+def get_classifier_training_job(exp_id, exp_version, state_name):
```

Comment:
@seanlip Right now, the get_multi cannot be used in practice. We are currently using get_job() inside create_mappings() method. We are using the get_job() method to retrieve the previous version's job and for this, wedont know the previous job's state_name until we actually check the change lists. So, we cannot call a get_multi() with all state_names as parameters. This could be used if we have all the state names of the jobs we want to retrieve. Am I missing something here?